tests/integration/pkg/client/client.go

```diff
@@ -101,3 +101,12 @@ func GetK8sClient() client.Client {
 
 	return c
 }
+
+func GetK8SConfig() (*rest.Config, error) {
+	config, err := loadKubeConfigOrDie()
```

Comment:
nit: is it really called `loadKubeConfigOrDie` and it doesn't actually kill the process?